ush/addtten.py

```diff
@@ -0,0 +1,20 @@
+#! /scratch1/BMC/wrfruc/Samuel.Trahan/soft/anaconda2-5.3.1/bin/python3.7
+
+import netCDF4, numpy, sys
+rootgrp=netCDF4.Dataset(sys.argv[1],'r+')
+
+T=rootgrp.variables['sphum']
```

Comment:
Please consider more descriptive variable names. T, v, and n  could be anything. 

We will also most likely want to use a contrib miniconda environment for retrieving an appropriate Python installation in case of @SamuelTrahanNOAA's eventual demise.

Reply:
I put this file in repository for record. We only need this once if we change the grid and will delete it after code update to add the tten directly in fortran executable.